appcenter-data/Data.d.ts

```diff
@@ -23,13 +23,13 @@ export class WriteOptions {
 }
 
 export interface DocumentWrapper {
-    deserializedValue: object | null;
-    eTag: string | null;
+    deserializedValue?: object;
```

Comment:
that's not the same: property being undefined or defined to be null.
did you check? I think we put null explicitly, please double check.

Reply:
Make sure you tested your assumption. 😄

Reply:
changed :)